test/files/pos/CharSequence-implicits.scala

```diff
@@ -0,0 +1,4 @@
+object Test extends App {
+  Array('a', 'b'): CharSequence
+  IndexedSeq('a', 'b'): CharSequence
+}
```

Comment:
Note that `private[scala]` is relatively (though not extremely) rare: 93 occurrences in `src`, fully half of which are in `src/reflect/scala/reflect/internal`. and in the case of this bug, `private[scala]` was only introduced by a mistake, then taken out again.